source/connect.txt

```diff
@@ -9,5 +9,147 @@ Connect to a Deployment
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
+
+
+Ensure that a MongoDB deployment is running before attempting to 
+connect to it with ``mongosh``. The connection method you use depends 
+on your deployment type.
+
+Local MongoDB Instance on Default Port
+--------------------------------------
+
+You can run ``mongosh`` without any command-line options
+to connect to a MongoDB instance running on your **localhost** with 
+**default port** 27017:
+
+.. code-block:: sh
+
+   mongosh
+
+Local MongoDB Instance on a Non-Default Port
+--------------------------------------------
+
+To specify a port to connect to on localhost, use the 
+:manual:`connection string </reference/connection-string>` format. For 
+example, to connect to a MongoDB instance running on localhost with a 
+non-default port 28015:
+
+.. code-block:: sh
+
+   mongosh "localhost:28015"
+
+MongoDB Instance on a Remote Host
+---------------------------------
+
+To specify a remote host and port, use a :manual:`connection string
```

Comment:
[optional] We can maybe highlight Atlas a bit more here. Consider one of:

- Making the title `MongoDB Instance on a Remote Host (Atlas)`

- Making the opening the sentence "To specify a remote host and port, such as an Atlas cluster, use a ..."

Reply:
I went back on this because I decided not to change the example to an Atlas example. Without this being at Atlas example it read strangely to me. It's still a possible that this should be Atlas focused, so if that's what we decided I'll make both changes together.